src/main/java/uk/co/ramp/event/ContactEventProcessor.java

```diff
@@ -100,12 +117,12 @@ private boolean isContactIsolated(
    * provided. Then it is automatically converted into exp(bias) = U/(1-U) based on Sigmoid(bias) =
    * U.
    *
+   * @param personA the most severe case
    * @param c
    * @param time
    * @return
    */
-  Optional<InfectionEvent> evaluateExposures(ContactEvent c, int time) {
-    Case personA = getMostSevere(population.get(c.to()), population.get(c.from()));
+  Optional<InfectionEvent> evaluateExposures(Case personA, ContactEvent c, int time) {
```

Comment:
It would be cleaner if we either pass in as args `Case victim, Case potentialInfector, int time` or `ContactEvent c, int time` instead of both personA and c, where c already contains information about personA.
For the latter option we could derive the victim/infector again using `getMostSevere`.

Reply:
No problem. Will change.